src/misc/lv_style.c

```diff
@@ -25,6 +25,101 @@
  *  GLOBAL VARIABLES
  **********************/
 
+uint8_t lv_style_lookup_table[256] = {
```

Comment:
What about making it `const` and using an other table in RAM (dynamically allocated?) for custom properties?

Reply:
Updated.